### ISSUE TITLE
Live location on map

### ISSUE THREAD

**Issue by rulkens** [theborderland/map#9]
Firstly, thanks so much for the placement team this year on adding so many useful features to the map! It's been really cool to see this progress so much and fast.

When I arrived at Borderland and was setting up camp, I realized that it would be useful to determine the camp boundaries when having the map open on my phone, using my real-time GPS location. Then I could just walk around and see if I were still within camp boundaries. I wanted to share it here, I might take up this task myself, but at least it's noted now :)

**Comment by lyret** [comment]
It would be good. A potential problem to handle is that the quality of GPS receivers vary a lot between phones - so it might cause confusions - would be good to clearly communicate that you might be 5-10 meters off from your actual location

**Comment by rulkens** [comment]
Yes, that makes sense. The Geolocation API also provides an accuracy parameter that could be used to show the borderling how accurate the measurement is (like how it's used in many apps):

https://developer.mozilla.org/en-US/docs/Web/API/GeolocationCoordinates

**Comment by Gusbod** [comment]
Yes this would be great! Actually the functionality is in there, it was just not activated. Last year it was tested and the reported accuracy was much worse than in say Google Maps. I didn't have time to investigate why though, as the data from the sensors should be the same. Anyways, lets fins a solution for next year! (as well as better caching...)

**Comment by fredrikelmegren** [comment]
I've been thinking about this for the last few years too.. but for some reason I keep forgetting about it after borderland 🤪. 

Re. the accuracy issue, I saw there's an option in the control "enableHighAccuracy". Not sure if that was enabled or even existed in 2022? (it's false by default)

Here's a PR: https://github.com/theborderland/map/pull/57
